client/settings/dpcs-settingspanel/settingspanel.py

```diff
@@ -0,0 +1,142 @@
+#coding: utf-8
```

Comment:
How about adding `#!/usr/bin/env python2` as the first line? Then, the panel can be launched as `./settingspanel.py` and the script above is not necessary.